test/performance/rfactor.cpp

```diff
@@ -0,0 +1,423 @@
+#include "Halide.h"
```

Comment:
It doesn't appear to be possible for this test to fail. I'd add some very conservative asserts on the improvement factors (e.g. check they don't make things substantially slower: improvement > 0.9) That should catch anything that breaks rfactor performance in some weird way.